src/data/screenshots_de.json

```diff
@@ -6,13 +6,18 @@
     },
     "section-main": {
         "headline": {
-            "title": "Machen Sie sich selbst ein Bild (Version 2.19)"
+            "title": "Machen Sie sich selbst ein Bild (Version 2.20)"
         },
         "content": {
             "textblock": [
-                "Verschaffen Sie sich mithilfe von Screenshots einen Überblick über die Funktionen, die die Corona-Warn-App mit Version 2.19 (verfügbar seit 16. März 2022) bietet. Bitte beachten Sie, dass die Screenshots für Android und iOS designbedingt leicht voneinander abweichen können.",
+                "Verschaffen Sie sich mithilfe von Screenshots einen Überblick über die Funktionen, die die Corona-Warn-App mit Version 2.20 (verfügbar seit ??. ????? 2022) bietet. Bitte beachten Sie, dass die Screenshots für Android und iOS designbedingt leicht voneinander abweichen können.",
                 "Mit diesem Update stellen wir neue und erweiterte Funktionen zur Verfügung:",
-                "<b>Zuordnung von Zertifikaten:</b> Die Zuordnung von Zertifikaten zu Personen wurde fehlertoleranter gestaltet. Kleine Unterschiede in der Schreibweise von Vor- und Nachnamen werden ignoriert, sodass die Zertifikate trotzdem derselben Person zugeordnet werden können.",
+                "<b>Anzeige erhöhten Risikos verkürzt:</b> Ein erhöhtes Risiko nach einer Risikobegegnung wird nur noch für 10 anstatt für 14 Tage angezeigt.",
+                "<b>Risikoanzeige nach positivem Testergebnis:</b> Auch wenn Sie einen positiven Test in der Corona-Warn-App registriert haben, wird Ihnen nun nach einer Risikobegegnung die rote Kachel für ein erhöhtes Risiko eingeblendet. Sieben Tage nach dem positiven Test wird ihr Risikostatus dauerhaft wieder eingeblendet.",
+                "<b>Nachricht zu neuer Risikobegegnung:</b> Wenn Sie nach einer Risikobegegnung eine weitere Begegnung mit erhöhtem Risiko hatten, dann wird Ihnen darüber jetzt eine Benachrichtigung angezeigt.",
+                "<b>Nachricht über Status-Änderung:</b> Wenn sich Ihr Status geändert hat (z. B. durch Hinzufügen eines Zertifikats, oder weil sich die Regeln geändert haben) werden Sie auf der Zertifikate-Registerkarte darüber informiert.",
```

Comment:
`z. B.` => `z.&nbsp;B.`

The space should be non-breaking.

Reply:
Fixed in https://github.com/corona-warn-app/cwa-website/pull/2543/commits/33ddb2e34ce36eb63feb55bd78cb292ca6c85272, thanks!